website/docs/cli/commands/terraform/terraform-workspace.mdx

```diff
@@ -49,7 +49,10 @@ atmos terraform workspace test/test-component-override-3 -s tenant1-ue2-dev
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------|:---------|
+| `--stack`             | Atmos stack                                                                                                                                                                                                                                                                                                                                                                                       | `-s`  | yes      |
+| `--dry-run`           | Dry run                                                                                                                                                                                                                                                                                                                                                                                           |       | no       |
+| `--process-templates` | Enable/disable Go template processing in Atmos stack manifests<br/>when executing the command.                                                                                                                                                                                                                                                                                                    |       | no       |
+| `--process-functions` | Enable/disable YAML functions processing in Atmos stack manifests<br/>when executing the command.                                                                                                                                                                                                                                                                                                 |       | no       |
+| `--skip`              | Skip processing a specific Atmos YAML function<br/>in Atmos stacks manifests when executing the command.<br/>To specify more than one function,<br/>use multiple `--skip` flags, or separate the functions with a comma:<br/>`atmos terraform workspace <component> -s <stack> --skip=eval --skip=include`<br/>`atmos terraform workspace <component> -s <stack> --skip=terraform.output,include` |       | no       |                                                 |       | no       |
```

Comment:
Please use definition lists instead of tables.

```
<dl>
  <dt>term</dt>
  <dd>definition and anything else</dd>
  <dt>term</dt>
  <dd>definition and anything else</dd>
  <dt>term</dt>
  <dd>definition and anything else</dd>
</dl>
```